orca/requester.py

```diff
@@ -1,48 +1,159 @@
 """Module responsible for handling requests to data servers"""
 
 import logging
+import requests
 from xarray import open_mfdataset, open_dataset
 import re
-from math import ceil
 import os
 from datetime import datetime
 
 
 logger = logging.getLogger("scripts")
 
 
-def file_from_opendap(url, outdir, outfile):
+def file_from_opendap(url, threshold, outdir, outfile):
     """Write to file from OPeNDAP link"""
-    urls = bisect_request(url)
+    input_dataset = open_dataset(url)
+    nbytes = input_dataset.nbytes / 2
+    data_vars = input_dataset.data_vars
+    data_vars_with_time = (
+        []
+    )  # Only use data variables with a time component for pivoting in bisect request.
```

Comment:
Is there a plain text overview anywhere in this repository of how requests are split and combined?

Reply:
There isn't, but yeah, that'd be good. The closest would be looking at the [bisect request](https://github.com/pacificclimate/orca/blob/i34-metadata-requests/tests/test_requester.py#L274) test and seeing what the input targets are and what the expected split would look like, and also seeing how they're [combined using xarray](https://github.com/pacificclimate/orca/blob/i34-metadata-requests/orca/requester.py#L36). Those two places would be good starting points for putting together a plain-language overview.